department-of-veterans-affairs/caseflow
# Locking hearing day

Resolves #7946 

### Description
This PR allows hearing coordinators to 'lock' a hearing day, meaning no more veterans can be scheduled.

It also includes some code cleanup. I've removed all functionality for updating VACOLS hearing days, updated some variable names to be more accurate, and added functionality for Caseflow video parent records.

### Testing Plan
1. Go to the daily docket for a Caseflow hearing day
1. Click 'Lock hearing day'
1. Confirm the hearing day does not show up on the Assign Hearings page or the dropdown on the Daily Docket
1. Go back to the daily docket
1. Click 'Unlock hearing day'
1. Confirm the hearing day shows up on the Assign Hearings page and the dropdown on the daily docket
